packages/bitcore-wallet-service/src/lib/expressapp.ts

```diff
@@ -1315,6 +1315,23 @@ export class ExpressApp {
       });
     });
 
+    router.post('/v1/service/checkAvailability', (req, res) => {
```

Comment:
I'd like to move away from promise chains and callbacks and instead use async/await syntax. Could you make this router function `async` and extend the try..catch below to encompass the whole function and `await server.checkServiceAvailability`?

Reply:
Fixed. I avoided using promises for this particular function